packages/server/src/question/question.entity.ts

```diff
@@ -65,4 +71,16 @@ export class QuestionModel extends BaseEntity {
 
   @Column({ nullable: true })
   isOnline: boolean;
+
+  /**
+   * Scopes
+   */
+  static openInQueue(queueId: number): SelectQueryBuilder<QuestionModel> {
```

Comment:
Inspired by Rails ActiveRecord scopes. This function gives you a QueryBuilder that has filters applied to give you questions that are open in a certain queue, so you can then join more stuff in, or just get the top 3, etc. If we make our queue order system more complex eventually (to allow manual reordering by TA's), we can just add that logic here!